cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -25,7 +25,6 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
```

Comment:
instead, the `"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"` can be removed.

Reply:
thanks for the suggestion, i have changed this.